fv3config/_datastore.py

```diff
@@ -22,6 +21,28 @@
 }
 
 
+def get_resolution(config):
```

Comment:
Was there a reason to move this from the public `derive` namespace to this private namespace? The idea of `fv3config.config.derive` (whose members should be imported to the top level instead, probably my bad) is to provide routines which derive data from a configuration dictionary (excluding data paths). If there isn't a need for it to be moved, I'd suggest keeping it in the module it was in.

Reply:
It was necessary to prevent a circular import issue. Currently, `get_resolution` is not imported by any `__init__`'s and is only used in `_datastore.py` so it seemed like that was a more appropriate place.

Reply:
OK, we can consider moving it later then.

Reply:
Sounds good.